mmv1/products/dns/terraform.yaml

```
@@ -64,7 +62,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       dnssecConfig.nonExistence: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
       id: !ruby/object:Overrides::Terraform::PropertyOverride
-        exclude: true
+        name: 'uniqueResourceID'
```

```suggestion
        name: 'managedZoneID'
```

I think we've used the resource name for these, typically!